server/db/migrate/20150408155047_remove_engine_hostname.rb

```diff
@@ -0,0 +1,9 @@
+class RenameEngineHostname < ActiveRecord::Migration
```

Comment:
@jmrodri, thanks you're right. I did it manually, since I couldn't do `rails g migration` within the plugin engine. How do you do it?